forgerock-core/src/main/java/org/forgerock/android/auth/biometric/BiometricAuth.kt

```diff
@@ -216,6 +219,40 @@ class BiometricAuth @JvmOverloads constructor(
 
     companion object {
         private val TAG = BiometricAuth::class.java.simpleName
+
+        @JvmStatic
+        fun isBiometricAvailable(applicationContext: Context) : Boolean{
+            var canAuthenticate = true
+            if (Build.VERSION.SDK_INT < 28) {
+                val keyguardManager : KeyguardManager = applicationContext.getSystemService(KEYGUARD_SERVICE) as KeyguardManager
+                val packageManager : PackageManager = applicationContext.packageManager
+                // Check if Fingerprint Sensor is supported
+                if(!packageManager.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT)) {
```

Comment:
same here if(packageManager.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT).not())